blender/arm/exporter.py

```diff
@@ -2152,7 +2152,7 @@ def execute(self):
 
         # No camera found
         if not self.camera_spawned:
-            log.warn('No camera found in active scene layers')
+            log.warn('Scene "'+self.scene.name+'" is missing a camera')
```

Comment:
I'd use `f` strings like you did above

Reply:
f-strings are also [quite fast](https://grski.pl/fstrings-performance.html) and in many cases faster than simple concatenation with `+`.